lib/avrora/config.ex

```diff
@@ -32,6 +32,7 @@ defmodule Avrora.Config do
   @callback registry_user_agent :: String.t() | nil
   @callback registry_ssl_cacerts :: binary() | nil
   @callback registry_ssl_cacert_path :: String.t() | nil
+  @callback registry_ssl_opts :: [:ssl.tls_option()] | nil
```

Comment:
Please also update documentation section above, highlighting that it will take precedence over the other SSL configuration options